src/App.js

```diff
@@ -109,6 +116,18 @@ class App extends Component {
     });
   }
 
+  updateInitiative(initiative, player) {
+    console.log('Set ' + player.title + ' initiative to ' + initiative);
+    player.initiative = initiative;
+    const playerStateIndex = this.state.players.find(
```

Comment:
It could be helpful to send this along when calling this method from `List` so you don't have to iterate over the whole object to find it, e.g. `updateInitiative(initiative, playerId)`.